src/Service/ElasticSearch.php

```diff
@@ -91,6 +91,14 @@ public function getClient(): Client
         return $this->client;
     }
 
+    /**
+     * @param array<array-key, mixed>|null $createOrUpdateBody
+     */
+    public function setElasticSearchIndex(string $indexName, array $createOrUpdateBody = null): Amp\Promise
+    {
+        return $this->client->createOrUpdateIndex($indexName, $createOrUpdateBody);
```

Comment:
@mmenozzi This method is only available from webgriffe/amp-elasticsearch 2.1.0, maybe we should adjust the constraint in composer.json?

Reply:
I rebased onto master which already have this constraint after merging #38.